### PR TITLE
iOS Documentation: Add Marker Example

### PR DESCRIPTION
**Pull request by louwers**
- Use DocC AutomaticArticleSubheading option to disable automatic 'Overview' heading https://www.swift.org/documentation/docc/automaticarticlesubheading
- Add marker example adapted from https://github.com/mapbox/ios-sdk-examples/blob/4fca7b8eb4004ded92ff5d9ed71f0f5f07b1f584/Examples/Swift/AddMarkerSymbolExample.swift
- Add a note to each example if it uses UIKit or SwiftUI. I am probably not going to add the same example for both UIKit and SwiftUI.

### CODE REVIEW

File: platform/ios/MapLibre.docc/MapLibre.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Powerful, free and open-source mapping toolkit with full control over data sourc
 - <doc:LocationPrivacyExample>
 - <doc:BlockingGesturesExample>
 - <doc:LineStyleLayerExampleUIKit>
+- <doc:WebAPIDataExampleUIKit>
```
Comment (HarelM) [inline platform/ios/MapLibre.docc/MapLibre.md:21]: Can this part be auto generated somehow? I see that you add a section here every time you add a new page...

File: platform/ios/MapLibre.docc/WebAPIDataExampleUIKit.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            guard let coor = item["coor"] as? [String: AnyObject],
+                  let point = coor["value"] as? String else { continue }
+
+            let parsedPoint = point.replacingOccurrences(of: "Point(", with: "").replacingOccurrences(of: ")", with: "")
+            let pointComponents = parsedPoint.components(separatedBy: " ")
+            let coordinate = CLLocationCoordinate2D(latitude: Double(pointComponents[1])!, longitude: Double(pointComponents[0])!)
+            let feature = MLNPointFeature()
+            feature.coordinate = coordinate
+            feature.title = title
+            // A feature’s attributes can used by runtime styling for things like text labels.
```
Comment (HarelM) [inline platform/ios/MapLibre.docc/WebAPIDataExampleUIKit.md:213]: I would consider making the examples a bit shorter, 200 line example is a bit long to read.

**Review by HarelM — APPROVED** [review]

File: platform/ios/MapLibre.docc/MapLibre.md (hunk 1 above)
Comment (louwers) [reply platform/ios/MapLibre.docc/MapLibre.md:21]: No, but I will manually sort the examples by category when I am done.

File: platform/ios/MapLibre.docc/WebAPIDataExampleUIKit.md (hunk 2 above)
Comment (louwers) [reply platform/ios/MapLibre.docc/WebAPIDataExampleUIKit.md:213]: I can do another pass and chop them up later.

**Pull request merged**